examples/gno.land/r/faucet/faucet.gno

```diff
@@ -9,60 +9,88 @@ import (
 
 var (
 	// configurable by admin.
-	gAdminAddr       std.Address = "g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq"
-	gControllerAddr  std.Address = "g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq"
-	gPerTransferSend             = std.Coins{std.Coin{"ugnot", 1000000}}
-	gInPause                     = false
-	gMessage                     = "# Community Faucet.\n\n"
+	gAdminAddr   std.Address    = "g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5"
+	gControllers [3]std.Address = [3]std.Address{} // limit it to 3
+
+	gInPause = false
+	gMessage = "# Community Faucet.\n\n"
 
 	// internal vars, for stats.
 	gTotalTransferred std.Coins
 	gTotalTransfers   = uint(0)
+
+	// per account request limit 350 gnot
```

Comment:
```suggestion
	// per request, limit 350 gnot.
```

can't see where you limit per account

Reply:
This is a per-account request because a request is from an account.  An account limit is implemented in the bot. It is in the community bot repo.